terraform/environments/core-vpc/vpc.tf

```diff
@@ -187,6 +187,28 @@ module "dns_zone_extend" {
   vpc_id      = module.vpc[each.key].vpc_id
   dns_domain  = ".modernisation-platform.internal"
 }
+#Locals needed for the private dns zone extend module
+locals {
+  private-application-zones = {
+      laa = "legalservices.interal",
+      hmpps = "hmpps.internal"
+    
+    }
+}
+
+
```

Comment:
Do we need the module if we're not creating any private zones that need associating across accounts?